clients/api/http/endpoints.go

```diff
@@ -117,12 +117,14 @@ func listClientsEndpoint(svc clients.Service) endpoint.Endpoint {
 
 		res := clientsPageRes{
 			clientsPageMetaRes: clientsPageMetaRes{
-				Total:  page.Total,
-				Offset: page.Offset,
-				Limit:  page.Limit,
+				Total: page.Total,
 			},
 			Clients: []viewClientRes{},
 		}
+		if !req.Page.OnlyTotal {
+			res.Offset = page.Offset
+			res.Limit = page.Limit
+		}
```

Comment:
Please revert back to original, JSON marshal will handle this case  We have added `omitempy` in JSON tags